rpi-eeprom-update

```diff
@@ -348,46 +469,66 @@ write_status_info()
    [ -z "${MACHINE_OUTPUT}" ] && return 0
 
    exit_code="${1}"
-   cur=${2}
-   new=${3}
+   bootloader_cur="${BOOTLOADER_CURRENT_VERSION}"
+   bootloader_new="${BOOTLOADER_LATEST_VERSION}"
+   vl805_cur="${VL805_CURRENT_VERSION}"
+   vl805_new="${VL805_LATEST_VERSION}"
 
     if [ "${JSON_OUTPUT}" = "no" ]; then
       cat > "${MACHINE_OUTPUT}" <<EOF
 EXITCODE="${exit_code}"
-CURRENT_TS=${cur}
-LATEST_TS=${new}
+BOOTLOADER_CURRENT=${bootloader_cur}
+BOOTLOADER_LATEST=${bootloader_new}
+VL805_CURRENT="${vl805_cur}"
+VL805_LATEST="${vl805_new}"
 EOF
    else
       cat > "${MACHINE_OUTPUT}" <<EOF
 {
   "EXITCODE": "${exit_code}",
-  "CURRENT_TS": ${cur},
-  "LATEST_TS": ${new}
+  "BOOTLOADER_CURRENT": ${bootloader_cur},
+  "BOOTLOADER_LATEST": ${bootloader_new}
```

Comment:
```suggestion
  "BOOTLOADER_LATEST": ${bootloader_new},
```

Reply:
Do you want all of the JSON values to default to zero instead of the empty string to ensure that the output is valid JSON ?

Reply:
If it's a string then an empty string (`""`) is fine, but for the integer time stamps JSON can't handle a missing value (as it was here, but probably because the wrong shell var had been used) so defaulting both to zero would be fine - I wasn't sure if `${bootloader_new}` could be blank (undefined). I suppose for consistency we should default both the JSON and non-JSON outputs to 0. Thanks!

Reply:
This is probably the easier change:
```
-  bootloader_cur="${BOOTLOADER_CURRENT_VERSION}"
-  bootloader_new="${BOOTLOADER_UPDATE_VERSION}"
+  bootloader_cur="${BOOTLOADER_CURRENT_VERSION:-0}"
+  bootloader_new="${BOOTLOADER_UPDATE_VERSION:-0}"
```
then just add the missing comma to the JSON output.